### DISCUSSION TITLE
My remix does not display accounts for each testing environment

### DISCUSSION THREAD

**Discussion by Apostleasd** [Cyfrin/foundry-full-course-cu#1596] [Q&A]
Why my remix does not display accounts for each testing environment
![1712851708283](https://github.com/Cyfrin/foundry-full-course-f23/assets/38981397/3cca5741-4bfa-4019-aae3-f3f5687c3964)
![1712851727310](https://github.com/Cyfrin/foundry-full-course-f23/assets/38981397/a1a14cca-7105-401e-976e-7ebce9bfe7a7)

**Comment by EngrPips** [comment]
Hello @Apostleasd , That most likely should be an internet connection issue, check your internet connection and then reload remix, it should load if your internet connection is stable.
